### DISCUSSION TITLE
Add in a toggle for container and container-fluid

### DISCUSSION THREAD

**Discussion by PizzaConsole** [oqtane/oqtane.framework#768] [Ideas]
Would it be of interest to anyone else to add in a toggle for the pages and module to utilize bootstrap's `container` vs `container-fluid`? I would really like to have this, but I do not want a whole new theme for it... Or I suppose there could be 2 Oqtane themes... 

It is prudent to note that the navbar is completely fluid

**Comment by sbwalker** [comment]
@PoisnFang can you please provide more detail of what you are referring to? Based on the description it sounds like this should be enabled through CSS rather than code.
